File: Topic Based Problems/01 Fundamentals/1 Beginner/14 Triangle Angle/Java/TriangleAngle.java
import java.util.Scanner;

public class TriangleAngle {
    public static void main(String[] args) {
        int firstAngle, secondAngle, thirdAngle;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter first angle: ");
        firstAngle = scanner.nextInt();

        System.out.print("Enter second angle: ");
        secondAngle = scanner.nextInt();

        thirdAngle = 180 - (firstAngle + secondAngle);

        System.out.println("Third angle is: " + thirdAngle);

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/14 Triangle Angle/C++/TriangleAngle.cpp
#include <iostream>

using namespace std;

int main() {
    int firstAngle, secondAngle, thirdAngle;

    cout << "Enter first angle: ";
    cin >> firstAngle;

    cout << "Enter second angle: ";
    cin >> secondAngle;

    thirdAngle = 180 - (firstAngle + secondAngle);

    cout << "Third angle is: " << thirdAngle << endl;

    return 0;
}